File: C_Programs/Enter10EmployeeSalaryAndDisplay.cpp
//wap to create a float array 0 10 elements to store salary of 10 employees

#include<stdio.h>
#include<conio.h>

int main()
{
	double salary[10];
	int cnt,a;
	for(cnt=0;cnt<10;cnt++)
	{
		printf("\nEnter the salary of %d : ",cnt);
		scanf("%lf",&salary[cnt]);
	}
	for(cnt=0;cnt<10;cnt++)
	{
		printf("\nSalary of [%d] = %.2lf",cnt,salary[cnt]);
	}
	return 0;
}

File: Core_Java_Programs/Enter10EmployeeSalaryAndDisplay.java
//wap to create a float array 0 10 elements to store salary of 10 employees
import java.util.*;

public class Enter10EmployeeSalaryAndDisplay {
    public static void main(String[] args) {
        float salary[]=new float[10];
        int cnt,a;
        Scanner sc=new Scanner(System.in);

        for(cnt=0;cnt<10;cnt++)
        {
            System.out.println("Enter the salary of "+cnt+" : ");
            salary[cnt]=sc.nextFloat();
        }

        for(cnt=0;cnt<10;cnt++)
        {
            System.out.println("Salary of "+cnt+" = "+salary[cnt]);
        }
    }
}
